src/config/context.rs

```diff
@@ -236,12 +236,12 @@ pub async fn build_context(cfg: &schema::SeafowlConfig) -> Result<SeafowlContext
     // (it will reload its schema before running the query)
 
     Ok(SeafowlContext {
+        config: Arc::new(cfg),
```

Comment:
Yeah, might as well make it an owned struct without the Arc

Reply:
I now remembered that I initially put the Arc on top of it due to the `scope_to_*` methods which create a new context on the heap, but it's probably ok to do it anyway since the config is tiny.